pyracf/common/invalid_segment_trait_error.py

```diff
@@ -0,0 +1,18 @@
+"""Exception to use when the user passes invalid segment-trait(s) in the traits dictionary."""
+
+
+class InvalidSegmentTraitError(Exception):
```

Comment:
Naming suggestion. I suggest changing the name of this exception to `SegmentTraitError`.

Reply:
Sure, Changed.